### PR TITLE
Create application-template.md

### PR DESCRIPTION
**Pull request by semuelle**
- Some clarifications and highlighting based on past issues,
- mention licensing,
- more emojis ✊,
- ask for 
  - short and long term plans,
  - community & contributors,
  - status of project before grant,
  - previous grant applications,
  - work done.

### CODE REVIEW

**Comment by Noc2** [comment]
Looks good to me. @mmagician should check if it causes any problems with the bots

**Comment by Noc2** [comment]
And @semuelle you need to sign the cla ;-)

**Review by alxs — COMMENTED** [review]
Nice, this is gonna spare us a lot of trouble. My review got a bit out of hand. Most amendments are unrelated to your changes I think.

File: applications/application-template.md [hunk 1]
```diff
@@ -59,34 +69,47 @@ Please describe the team's relevant experience.  If the project involves develop
 * https://www.linkedin.com/<person_1>
 * https://www.linkedin.com/<person_2>
 
+## Development Status :open_book: 
+
+If you already started implementing your project or the project is part of a larger repository, please provide a link and a description of the code here. In any case, please provide some documentation on the research and other work you have conducted before applying. This could be:
+
+- links to improvement proposals or [RFPs](https://github.com/w3f/General-Grants-Program/tree/master/rfp-proposal) (request for proposals),
```
[the remark this answers is not in the document]
Comment (semuelle) [reply applications/application-template.md:76]: I would have bet the plural of RFP is "request for proposals", but you're right.

File: applications/application-template.md [hunk 2]
```diff
@@ -59,34 +69,47 @@ Please describe the team's relevant experience.  If the project involves develop
...
+* include total amount of funding requested per milestone.
+* include documentation (tutorials, API specifications, architecture diagrams, whatever is appropriate) in each milestone. This ensures that the code can be widely used by the community.
+* provide a test suite, comprising unit and integration tests, along with a guide on how to set up and run them.
+* commit to providing dockerfiles for the delivery of your project. 
+* indicate the milestone duration, as well as number of Full-Time Employees working on each milestone, and include the approximate number of days along with cost per day.
+* _Deliverables 0a-0d are mandatory_ and should not be removed, unless you explicitly specify a reason within the PR's `Additional Notes` section (e.g. Milestone X is research oriented and as such there is no code to test)
+
+> :zap: If any of your deliverables is based on somebody else's work, make sure you work and publish _under the terms of the license_ and that you **highlight this fact in your milestone documentation** and in the source code if applicable!
```
[the remark this answers is not in the document]
Comment (semuelle) [reply applications/application-template.md:99]: taking it to the next level, Alex. :) Fair game, though.

**Pull request merged**